examples/pytorch/graphsage/node_classification.py

```diff
@@ -72,9 +72,9 @@ def evaluate(model, graph, dataloader):
 def layerwise_infer(device, graph, nid, model, batch_size):
     model.eval()
     with torch.no_grad():
-        pred = model.inference(graph, device, batch_size).to(device)
```

Comment:
Should we remove the `device` parameter then?

Reply:
`device` is still needed to pass into `model.inference()` function.